projects/questionset-editor-library/src/lib/components/meta-form/meta-form.component.ts

```diff
@@ -214,7 +214,7 @@ export class MetaFormComponent implements OnChanges, OnDestroy {
           field.default = _.get(metaDataFields, 'recordedBy') !== 'Self' ? 'Yes' : 'No' ;
         }
         if (field.code === 'mode') {
-          field.default = _.get(metaDataFields, 'eval.mode')
+          field.default = _.get(metaDataFields, 'evalMode') == this.configService.editorConfig.evalMode || _.get(metaDataFields, 'mode') === true ? true : false;
```

Comment:
@pavankumar0408 Instead of `field.code === 'mode'` can we use different name for this field say `serverMode`.
Editor config have a mode property and it's creating confusion at a first look what this mode is. Please provide a meaningful name to this field.

Reply:
done changed to serverMode